src/components/Models/AnimationModel/AnimationModel.component.tsx

```diff
@@ -37,22 +37,22 @@ export const AnimationModel: FC<AnimationModelProps> = ({
   const { scene } = useGltfLoader(modelSrc);
   const { nodes } = useGraph(scene);
 
+  const animationClip = useMemo(async () => loadAnimationClip(animationSrc), [animationSrc]);
```

Comment:
all animations are now memoized because we saw when requesting multiple times different avatars always request for animationClip file 

![image](https://github.com/readyplayerme/visage/assets/18527021/4d9760df-8416-4360-aa7d-e9166ea00e7e)